index.js

```diff
@@ -43,9 +43,17 @@ async function subsetFont(
   exports.hb_blob_destroy(blob);
 
   const input = exports.hb_subset_input_create_or_fail();
+  if (input === 0) {
+    throw new Error(
```

Comment:
This also can benefit those memory cleanups just for the sake completeness, thanks you very much :)

Reply:
Oh yeah, moved some stuff around so the other things get allocated after `input` instead: 2e3baa2